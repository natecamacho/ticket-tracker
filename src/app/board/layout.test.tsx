import { render } from "@testing-library/react";
import Layout from "./layout";

describe("Board Layout component", () => {
  it("should render Board Layout unchanged", () => {
    const { container } = render(
      <Layout>
        <div>Test</div>
      </Layout>
    );

    expect(container).toMatchSnapshot();
  });
});
