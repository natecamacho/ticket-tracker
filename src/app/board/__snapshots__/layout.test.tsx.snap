// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board Layout component should render Board Layout unchanged 1`] = `
<div>
  <div
    class="flex h-screen flex-col md:flex-row md:overflow-hidden"
  >
    <div
      class="w-full flex-none md:w-64"
    >
      <div
        class="flex h-full flex-col px-2 py-4"
      >
        <a
          class="mb-2 flex h-40 items-end justify-start rounded-md bg-blue-600 p-4"
          href="/"
        >
          Nate's Ticket Tracker
        </a>
        <div
          class="flex grow flex-col justify-between space-x-0 space-y-2"
        >
          <a
            class="flex h-[48px] flex-none justify-center p-2 px-3 items-center gap-2 rounded-md bg-gray-50 text-sm font-medium hover:bg-sky-100 hover:text-blue-600"
            href="/board"
          >
            <p
              class="block"
            >
              Board
            </p>
          </a>
          <a
            class="flex h-[48px] flex-none justify-center p-2 px-3 items-center gap-2 rounded-md bg-gray-50 text-sm font-medium hover:bg-sky-100 hover:text-blue-600"
            href="/tickets"
          >
            <p
              class="block"
            >
              Tickets
            </p>
          </a>
          <a
            class="flex h-[48px] flex-none justify-center p-2 px-3 items-center gap-2 rounded-md bg-gray-50 text-sm font-medium hover:bg-sky-100 hover:text-blue-600"
            href="/backlog"
          >
            <p
              class="block"
            >
              Backlog
            </p>
          </a>
          <div
            class="block h-auto w-full grow rounded-md bg-gray-50"
          />
        </div>
      </div>
    </div>
    <div
      class="flex-grow p-6 md:overflow-y-auto md:p-12"
    >
      <div>
        Test
      </div>
    </div>
  </div>
</div>
`;
